sycl/test-e2e/Matrix/joint_matrix_bf16_fill_k_cache_arg_dim.cpp

```diff
@@ -6,7 +6,8 @@
 //
 //===----------------------------------------------------------------------===//
 // REQUIRES: aspect-ext_intel_matrix
-
+// XFAIL: arch-intel_gpu_bmg_g21
+// XFAIL-TRACKER: https://github.com/intel/llvm/issues/16922
```

Comment:
Could you please move it to the other xfails in this test?

Reply:
fixed in latest commit